themes/beaver/layouts/partials/blog/header.html

```diff
@@ -0,0 +1,94 @@
+
+<meta property="og:locale" content="en_US" />
+<meta property="og:title" content="{{ .Title }} | JetThoughts" />
+<meta property="og:description" content="{{ .Params.description }}" />
+<meta property="og:url" content="{{ .Permalink }}" />
+<meta property="og:site_name" content="JetThoughts" />
+<meta property="og:updated_time" content="{{ .Lastmod.Format " 2006-01-02T15:04:05-07:00" }}" />
+
+<meta name="twitter:title" content="{{ .Title}} | JetThoughts" />
+<meta name="twitter:site" content="@jetthoughts" />
+<meta name="twitter:creator" content="@jetthoughts" />
+
+{{ if eq .Section "blog" }}
+{{ with .Params.created_at }}
+<meta property="article:published_time" content="{{ . | time.Format " 2006-01-02T15:04:05Z07:00" }}">
+{{ end }}
+<meta property="article:modified_time" content="{{ .Lastmod | time.Format " 2006-01-02T15:04:05Z07:00" }}">
+{{ end }}
+
+{{ $url := .RelPermalink }}
+{{ if or (eq $url "/blog/") (strings.HasPrefix $url "/blog/page/") }}
+<meta name="robots" content="NOODP">
+{{ end }}
+
+{{ with .Page.Resources.GetMatch .Page.Params.metatags.image }}
+<meta property="og:image" content="{{ .Permalink }}" />
+<meta property="og:image:width" content="{{ .Width }}" />
+<meta property="og:image:height" content="{{ .Height }}" />
+<meta name="twitter:image" content="{{ .Permalink }}" />
+<meta name="twitter:card" content="summary_large_image">
+{{ end }}
+
+{{ if eq .Type "tags" }}
+{{ $blogIndex := .Site.GetPage "blog" }} <!-- Get the blog index page -->
+{{ with $blogIndex.Resources.GetMatch $blogIndex.Params.metatags.image }}
+<meta property="og:image" content="{{ .Permalink }}" />
+<meta property="og:image:width" content="{{ .Width }}" />
+<meta property="og:image:height" content="{{ .Height }}" />
+<meta name="twitter:image" content="{{ .Permalink }}" />
+<meta name="twitter:card" content="summary_large_image">
+{{ end }}
+{{ end }}
+
+<link rel="stylesheet" href="/css/newblog.css">
+
+</head>
+
+<body>
+
+
+    <header class="header">
+        <nav class="navbar">
+            <a class="logo" href="/" title="JetThoughts">
+                {{ partial "svg" "jetthoughts" }}
+            </a>
+
+            <input type="checkbox" id="menu-toggle" class="menu-toggle">
+            <label for="menu-toggle" class="menu-icon">
+                <div class="bar"></div>
+                <div class="bar"></div>
+                <div class="bar"></div>
+            </label>
+
+            <div class="nav-links">
+                {{ range .Site.Menus.main }}
+                {{ if eq .URL "/contact-us/" }}
+                <a class="btn" href="{{ .URL }}">{{ .Name }}</a>
+                {{ else }}
+                <a class="nav-item {{ if eq .URL $.RelPermalink }}active{{ end }}" href="{{ .URL }}">{{ .Name }}</a>
+                {{ end }}
+                {{ end }}
+            </div>
+        </nav>
+    </header>
+
+    <div class="wrapper">
```

Comment:
we should not break tree structure in partials, each partial should have only one root element

Reply:
In hugo you can do both way.

Converted posts and post list to use baseof